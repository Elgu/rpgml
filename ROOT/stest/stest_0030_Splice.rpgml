
Splice s( dims=2, nx=2, ny=3 );

float[,] a = [ 1.0f, 2.0f; 3.0f, 4.0f ];

for x = 0 to 2-1
for y = 0 to 3-1
{
  Output( a ) + x*10.0f + y*100.0f -> s.in[ x, y ];
}

print( s.out );
print( "\n" );

exit();

