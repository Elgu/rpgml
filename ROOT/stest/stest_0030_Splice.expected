[ 1, 11, 2, 12; 101, 111, 102, 112; 201, 211, 202, 212; 3, 13, 4, 14; 103, 113, 104, 114; 203, 213, 204, 214 ]
